chapters/en/slides/module2/module2_01.Rmd

```diff
@@ -29,34 +29,6 @@ Notes:
 
 ---
 
-```{python include=FALSE}
-import altair as alt
-from vega_datasets import data
-
-
-cars = data.cars()
-alt.Chart(cars).mark_point().encode(
-    x='Weight_in_lbs',
-    y='Miles_per_Gallon',
-    color='Origin')
-```
-
-
-```{python}
-point_plot = alt.Chart(cars).mark_point().encode(
-    x='Weight_in_lbs',
-    y='Miles_per_Gallon',
-    color='Origin')
-    
-line_plot = alt.Chart(cars).mark_line().encode(
-    x='Year',
-    y='mean(Miles_per_Gallon)',
-    color='Origin')
-    
-point_plot | line_plot 
-```
-
```

Comment:
Were you suppose to delete this or was it accidental?

Reply:
Intentional. This just looks like a section where I was trying out things, it has nothing to do with the notes, which should be on the previous page.

Reply:
Shouldn't we update the md file then? 
And what do you want instead of the plots then?

Reply:
I don't think this is in the md file, do you see it there? The notes should go on the previous slide, so there is nothing here.

Reply:
It currently looks like this from the deploy preview which I'm going to use. 
![image](https://user-images.githubusercontent.com/42727648/110536671-953b9a00-80d6-11eb-9fc2-ea0ca2cd3a03.png)

Reply:
Everything else is find and I'll merge to master.

Reply:
I'll check that but merge for now.